azext_iot/_help.py

```diff
@@ -764,6 +764,11 @@
         --content '{\\"moduleContent\\": {\\"properties.desired.chillerWaterSettings\\": {\\"temperature\\": 38, \\"pressure\\": 78}}}'
         --target-condition "from devices.modules where tags.building=9" --priority 1
         --metrics '{\\"metrics\\": {\\"queries\\": {\\"mymetric\\":\\"select moduleId from devices.modules where tags.location=''US''\\"}}}'
+    - name: Create a device configuration with an alternative input style of labels and metrics (shell agnostic)
```

Comment:
nit to add '.' / period to the example names (more like example sentence)

Reply:
got it thanks!